families/supplychain/postgreSQL/tables_sample_data.sql

```diff
@@ -0,0 +1,100 @@
+-- Definition of tables for Supply Chain DB
+-- Tables to create
+
+
+CREATE TABLE block (
+	block_id			char(128) primary key,
+	block_num			integer, 
+	state_root_hash		char(64)
+);
+
+
+CREATE TABLE record (
+	id 					bigserial primary key,
+	start_block_num		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	end_block_num		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	identifier 			varchar(128), 	-- The natural key of the record, serial number of attached sensor identifier.
+	creation_time 		bigint,			-- The time the record was created
+	finalized 			boolean			-- Is the record finalized?
+);
+
+
+CREATE TABLE agent(
+	id 					bigserial primary key,		-- primary key
+	start_block_num 	integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	end_block_num 		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	identifier  		varchar(128),	-- Hex-encoded public key of agent
+	name 				text			-- The agent name
+);
+
+CREATE TABLE type_enum(
+	id 					integer primary key, 		-- primary key
+	name				char(12), 		-- OWNER, CUSTODIAN
+	code  				smallint		-- 0, 1
+);
+
+
+CREATE TABLE status_enum(
+	id 					integer primary key, 		--- primary key
+	name				char(12),		-- OPEN, CANCELED, REJECTED, ACCEPTED
+	code 				smallint 		-- 0, 1, 2, 3		
+);
+
+CREATE TABLE record_agent(
+	id 					bigserial primary key, 						-- primary key
+	record_id			bigserial references record(id),	-- Foreign key from record/id
+	agent_identifier	varchar(128),					-- This field is loosely coupled to agent/identifier
+	start_time			bigint,
+	agent_type			integer	 references type_enum(id)	-- Fore\ign key from type_enum/id
+);
+
+
+CREATE TABLE application(
+	id 					bigserial primary key, 		-- primary key
+	start_block_num		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	end_block_num		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	record_identifier	varchar(128),	-- Loosely coupled record/identifier
+	applicant 			varchar(128),			-- Public key of the applicant
+	creation_time		bigint,
+	type 				integer references type_enum(id),		-- Foreign key from type_enum/id
+	status 				integer references status_enum(id),		-- Foreign key from status_enum/id
+	terms				text
+);
+
+
+
+
+
+-- SAMPLE DATA BELOW
+
+INSERT INTO type_enum VALUES(1, 'OWNER', 1), (2, 'CUSTODIAN', 2);
```

Comment:
You change this in the opposite way. The primary key should match the enum value, not the other way round:

```
(0, 'OWNER', 0), (1, 'CUSTODIAN', 1)
```

The same is true for the status enum